README.md

```diff
@@ -65,12 +65,15 @@ tcp_receiver
 tcp_sender -req=ex -cmd=send -host=localhost:20249
 ```
 
-## CDAQ evio example analysis
+## Run flags examples
+
+### CDAQ evio example analysis
 
 
 Run parameters:
 
 ```
+jana4ml4fpga
 -Pplugins=log,root_output,CDAQfile,example_evio_analysis
 -Pjana:debug_plugin_loading=1
 -PSingleEvioEventFileSource:LogLevel=trace
```

Comment:
This should be `-PCDAQEVIOFileSource:LogLevel=trace` since I do not see your single_event_evio plugin.